Editor/Resources/EditorWindow/Components/StatusIndicator.uss

```diff
@@ -0,0 +1,32 @@
+﻿/* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. */
+/* SPDX-License-Identifier: Apache-2.0 */
+
+@import '../common.uss';
+
+.status-indicator--in-progress {
+    color: var(--color-font-link);
```

Comment:
minor: do you need a style for inactive as well?

Reply:
No as the style matches the default as imported in common.uss